deeptools/_version.py

```diff
@@ -2,4 +2,4 @@
 # This file is originally generated from Git information by running 'setup.py
 # version'. Distribution tarballs contain a pre-generated copy of this file.
 
-__version__ = '1.6.0'
+__version__ = '1.6.0-173-gc811461'
```

Comment:
This is how it works: whenever you run setup.py the version number is automatically updated to match the commit hash. This is only relevant for the installation and should not be push to the repository.